tests/product/tests/data/process/process_test_cases.py

```diff
@@ -1344,8 +1343,10 @@
     *cis_1_2_29,
     *skip_param_case(skip_list=[*cis_1_2_32],
                      data_to_report=SkipReportData(
-                         skip_reason="This case fails and breaks cluster")
-                     )
+                         url_title="security-team: #4981",
+                         url_link="https://github.com/elastic/security-team/issues/4981",
+                         skip_reason="Depends on this issue to be closed before fixing"
```

Comment:
the issue is closed, plus per your [comment](https://github.com/elastic/security-team/issues/4193#issuecomment-1281924261):

> CIS 1.2.32: K8S won't accept dummy values for --tls-cipher-suites; valid cipher suites have to be provided.

So we is it on skip?